homeassistant/components/flexpool/sensor.py

```diff
@@ -0,0 +1,69 @@
+"""Support for Etherscan sensors."""
+from datetime import timedelta
+
+import flexpoolapi
+
+from .sensors import (
+    FlexpoolBalanceSensor,
+    FlexpoolHashrateSensor,
+    FlexpoolPoolHashrateSensor,
+    FlexpoolPoolLuckSensor,
+    FlexpoolPoolWorkersSensor,
+    FlexpoolWorkerHashrateSensor,
+    FlexpoolWorkerShareSensor,
+)
+
+SCAN_INTERVAL = timedelta(minutes=5)
+
+
+async def async_setup_entry(hass, entry, async_add_entities, discovery_info=None):
```

Comment:
`discovery_info` is not part of this signature

```suggestion
async def async_setup_entry(hass, entry, async_add_entities):
```